src/Word/CamelCaseToDash.php

```diff
@@ -16,6 +16,6 @@ final class CamelCaseToDash extends CamelCaseToSeparator
 {
     public function __construct()
```

Comment:
Perhaps drop this constructor entirely, so that we just end up with

```php
public function filter(mixed $value): mixed {
    $filter = new CamelCaseToSeparator(['separator' => '-']);
    
    return $filter->filter($value);
}
```

… and of course the required `__invoke` method and removal of  inheritance